frontend/app/layout.js

```diff
@@ -1,27 +1,32 @@
-import localFont from 'next/font/local';
-import './globals.css';
-import { SessionProvider } from 'next-auth/react';
+import localFont from "next/font/local";
+import "./_styles/globals.css";
+import { SessionProvider } from "next-auth/react";
 
 const geistSans = localFont({
-  src: './fonts/GeistVF.woff',
-  variable: '--font-geist-sans',
-  weight: '100 900',
+  src: "./fonts/GeistVF.woff",
```

Comment:
this seems like changes I already seen in some other PR of yours. Please make sure that your PRs only contain the new code that changes. Take a look into rebasing functionality of git for that :)

Reply:
this is not my code, I just change meta data